app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError, \
+                        OutdatedVaccineError, \
+                        NotWearingMaskError
+import datetime
```

Comment:
standard library imports should come before application-specific imports. See [pep8](https://peps.python.org/pep-0008/#imports)